### PR TITLE
post publish using tag from the package version

### LINKED ISSUE #5465: Use publish tag argument in post-publish
While publishing the release candidate (rc) for v2, I noticed this line should change to avoid publishing dist and locales over the tag `latest`.

https://github.com/plotly/plotly.js/blob/90ec21b92b4d3d4ac6a15dc12d9fe13046c55ac5/tasks/sync_packages.js#L309

I made the temporary change to `npm publish --tag rc` at the publish time; however it would be nice to fix this before the next release.

### PR DESCRIPTION
**Pull request by archmoj**
Fixes #5465.

@plotly/plotly_js

### CODE REVIEW

File: tasks/sync_packages.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ var glob = require('glob');
 var common = require('./util/common');
 var constants = require('./util/constants');
 var pkg = require('../package.json');
+var rc = pkg.version.split('-')[1];
```
Comment (nicolaskruchten) [inline tasks/sync_packages.js:10]: won't this error out if it's a non-RC version string like `2.0.1` ?

Comment (archmoj) [reply tasks/sync_packages.js:10]: No I don't think so. It was tested. In that case the tag and variable `rc` would be undefined.

Comment (nicolaskruchten) [reply tasks/sync_packages.js:10]: Oh Javascript :) `"string".split("-")[1] -> undefined` ... ok then! I've been doing Python too long!

**Review by alexcjohnson — APPROVED** [review]
💃

**Pull request merged**